### DISCUSSION TITLE
how to smooth the border of mesh

### DISCUSSION THREAD

**Discussion by SivHMA** [marcomusy/vedo#684] [Q&A]
I have a mesh with rough boundaries. whether any function that can smooth the border in vedo?

![image](https://user-images.githubusercontent.com/29498630/186890632-563fc56e-676f-41d9-b9d7-1c22f51aca2f.png)

**Comment by marcomusy** [comment]
hi, have you tried with
`my_mesh.smooth(boundary=True)`
